README : gradle metadata is no more required (#121)
benasher44/uuid: README.md

- Add `mavenCentral()` to your repositories
- Add `implementation "com.benasher44:uuid:<version>"` as a dependency in your `commonMain` `sourceSets`.

> This library publishes gradle module metadata. Nothing to do if you use Gradle 6+, but if not, you should have `enableFeaturePreview("GRADLE_METADATA")` in your settings.gradle(.kts).

Reviewer: ```suggestion
This library publishes gradle module metadata. If you're using Gradle prior to version 6, you should have `enableFeaturePreview("GRADLE_METADATA")` in your settings.gradle(.kts).
```
